src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -851,6 +851,91 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Status of the transparent data encryption.',
                    arg_type=get_enum_type(TransparentDataEncryptionStatus))
 
+    ###############################################
+    #                sql db ltr                   #
+    ###############################################
+    with self.argument_context('sql db ltr-policy set') as c:
+        create_args_for_complex_type(
+            c, 'parameters', Database, [
+                'weekly_retention',
+                'monthly_retention',
+                'yearly_retention',
+                'week_of_year'])
+
+        c.argument('weekly_retention',
+                   help='Retention for the weekly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
+
+        c.argument('monthly_retention',
+                   help='Retention for the monthly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
+
+        c.argument('yearly_retention',
+                   help='Retention for the yearly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
+
+        c.argument('week_of_year',
+                   help='The Week of Year, 1 to 52, in which to take the yearly LTR backup.')
+
+    with self.argument_context('sql db ltr-backup') as c:
+        c.argument('location_name',
+                   required=True,
+                   arg_type=get_location_type(self.cli_ctx),
+                   help='The location of the desired backups.')
+
+        c.argument('backup_name',
+                   options_list=['--name', '-n'],
+                   help='The name of the LTR backup. '
+                   'Use \'az sql db ltr-backup show\' or \'az sql db ltr-backup list\' for backup name.')
+
+        c.argument('long_term_retention_server_name',
+                   options_list=['--server-name', '--server', '-s'],
```

Comment:
Why --server-name? --server -s should be the only options imo

Reply:
Saw this done somewhere else, and I wanted to be consistent.  Can easily change to just --server -s